styleguide/config/config.yml

```diff
@@ -18,7 +18,7 @@ cacheBusterOn: 'true'
 exportDir: export
 publicDir: public
 sourceDir: source
-defaultPattern: 'pages-style-guide-welcome'
+defaultPattern: 'base-style-guide-welcome'
```

Comment:
This shouldn't show in the navigation anywhere. Please see Pattern Lab documentation here - http://patternlab.io/docs/pattern-hiding.html

Reply:
@labbydev Pattern hiding will prevent that pattern from being accessible via any url. The use case for this sort of pattern hiding is if you have atoms that you will include in a molecule, but do not want viewable on its own. Based on my tests and their documentation, this will not work when attempting to use for the default pattern.